vite.config.js

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
   plugins: [react()],
   publicDir: false, // This feature is deactivated because the 'public' dir also has a special meaning for the Play framework
   build: {
+    sourcemap: "inline",
```

Comment:
What's this for?

Reply:
Just useful for local dev: https://github.com/guardian/media-atom-maker/issues/931